apps/cowswap-frontend/src/common/updaters/orders/utils.ts

```diff
@@ -74,53 +74,29 @@ export function computeOrderSummary({
 
 type PopupData = {
   status: OrderTransitionStatus
-  popupData?: OrderLogPopupMixData
+  order: EnrichedOrder
 }
 
 export async function fetchOrderPopupData(orderFromStore: Order, chainId: ChainId): Promise<PopupData | null> {
   // Skip EthFlow creating orders
   if (orderFromStore.status === OrderStatus.CREATING) {
     return null
   }
-  // Get order from API
-  let orderFromApi: EnrichedOrder | null = null
+
   try {
     const isComposableCowChildOrder = getIsComposableCowChildOrder(orderFromStore)
     // For ComposableCow child orders always request PROD order-book
-    orderFromApi = await getOrder(chainId, orderFromStore.id, isComposableCowChildOrder ? 'prod' : undefined)
+    const order = await getOrder(chainId, orderFromStore.id, isComposableCowChildOrder ? 'prod' : undefined)
+
+    if (!order) return null
+
+    const status = classifyOrder(order)
+
+    return { status, order }
   } catch (e: any) {
     console.debug(
       `[PendingOrdersUpdater] Failed to fetch order popup data on chain ${chainId} for order ${orderFromStore.id}`
     )
     return null
   }
-  const status = classifyOrder(orderFromApi)
-
-  let popupData = undefined
-
-  switch (status) {
-    case 'fulfilled':
-      popupData = {
-        id: orderFromStore.id,
-        fulfillmentTime: new Date().toISOString(),
-        transactionHash: '', // there's no need  for a txHash as we'll link the notification to the Explorer
-        summary: computeOrderSummary({ orderFromStore, orderFromApi }),
```

Comment:
We don't need `summary` anymore, since we compute it in `OrdersNotificationsUpdater`